android/src/main/java/playground/android/model.kt

```diff
@@ -0,0 +1,12 @@
+package playground.android
+
+data class LibraryName(var name:String)
```

Comment:
I would prefer if there was a second parameter: 
`val createIntent: (Context) -> Intent`

This way adding a library could be done at only one place in `Suppliers.libraries`, without the need to modify the recycler adapter

Reply:
okay, I will do that